pyblish/cli.py

```diff
@@ -303,4 +334,42 @@ def publish(ctx,
         click.echo(_format_time(_start, _end))
 
 
+@click.command()
+@click.argument("package", default="pyblish_qml")
+@click.pass_context
+def gui(ctx, package):
+
+    environ = os.environ.copy()
+    context = ctx.obj["context"]
+
+    with _cli_plugin(data=context.data) as plugin_path:
+        environ["PYBLISHPLUGINPATH"] = os.pathsep.join(
+            ctx.obj["plugin_paths"] + [plugin_path]
+        )
+
+        process = subprocess.Popen(
+            [sys.executable, "-m", package + ".__main__"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=environ,
+            bufsize=1,
+            universal_newlines=True
+        )
+        while True:
```

Comment:
Listening to `stdout` is tricky. All sorts of things can be printed from the host, and some things differ across Python 2 and 3.

I can't say for sure whether the approach you have here will *not* work without issues, but I *can* say that the below example *would* work. If you don't mind, I'd like to see this in there instead.

```python
for line in iter(process.stdout.readline, b""):
  ...
```